app/views/conferences_primer_theme/pages/homepage.html.haml

```diff
@@ -27,5 +27,8 @@
           - if current_conference.present? && current_conference.finish_date >= Date.today
             = link_to t('.more_info'), frontend_conference_path(current_conference),
                       class: %w[btn-mktg btn-large-mktg btn-primary-mktg f3 mt-4]
+          - elsif ::Spina::Page.where(view_template: 'about').any?
```

Comment:
this isn't exactly a robust way of doing this - I suppose the alternative is to have another part on the homepage which I could implement if you think this is too brittle.

Reply:
Actually since the about page as defined in the theme as undeletable I think it would be okay to error if it doesn't exist.
https://github.com/jmalcic/spina-conferences-primer_theme/blob/ae3fcd8cab04d7b01aa16008c27b098747f3ec3b/config/initializers/themes/conferences_primer_theme.rb#L199-L204
```suggestion
          - else
```